src/Core/NotificationMenu.elm

```diff
@@ -69,14 +69,23 @@ update msg model =
 -}
 view : Model -> Html Msg
 view model =
-    label
-        [ class "main-header-menu-link main-header-menu-link--extra-padded"
-        , for "notifications-drawer-open"
-        , onClick ClearBadge
-        ]
-        [ badgeView model.unseenCount
-        , i [ class "icon" ] [ icon "bell" { size = "small", colour = "white", colouring = "outline" } ]
-        ]
+    let
+        unseenCount =
+            (toString (Maybe.withDefault 0 model.unseenCount))
+    in
+        label
+            [ class "main-header-menu-link main-header-menu-link--extra-padded"
+            , for "notifications-drawer-open"
+            , onClick ClearBadge
+            , attribute "data-interaction-section" "notification menu"
+            , attribute "data-interaction-type" "open modal"
```

Comment:
we already track elements which `data-interaction-type` is `open modal` so I've decided to use the same binding to keep it generic. if preferred I can create a new one like `open notification drawer`.

Reply:
Good idea!